ddmark/validation_unit_test.go

```diff
@@ -0,0 +1,142 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package ddmark_test
```

Comment:
Would it be alright if I pushed a commit with a few more ideas for unit tests in this file? Everything already here lgtm

Reply:
Please do ! I can easily imagine there's a few bits missing somewhere